adoc/advanced_topics_image_management.adoc

```diff
@@ -121,6 +121,8 @@ salt '$your_minion' state.highstate
 
 Create an activation key associated with the channel that your images will use.
 
+NOTE: only activation keys associated with a channel that is not "SUSE Manager Default" can be used to build Containers.
```

Comment:
NOTE: To build containers, you will need an activation key that is associated with a channel other than "SUSE Manager Default".

Reply:
Additionally, this should probably be in an admonition. Let me know if you want me to do the changes.